request.go

```diff
@@ -297,6 +297,31 @@ func (r *Request) SetMultipartField(param, fileName, contentType string, reader
 	return r
 }
 
+// SetMultipartFields method is to set multiple data fields using io.Reader for multipart upload.
+// Example: set multiple json strings.
+//  fields := []*resty.MultipartField{
+//		&resty.MultipartField{
+//			Param:       "uploadManifest1",
+//			FileName:    "upload-file-1.json",
+//			ContentType: "application/json",
+//			Reader:      strings.NewReader(`{"input": {"name": "Uploaded document 1", "_filename" : ["file1.txt"]}}`),
+//		},
+//		&resty.MultipartField{
+//			Param:       "uploadManifest2",
+//			FileName:    "upload-file-2.json",
+//			ContentType: "application/json",
+//			Reader:      strings.NewReader(`{"input": {"name": "Uploaded document 2", "_filename" : ["file2.txt"]}}`),
+//		},
+//	}
+//  resty.R().SetMultipartFields(fields)
+func (r *Request) SetMultipartFields(fields []*MultipartField) *Request {
```

Comment:
@e11ni Could you change the func to `variadic` func `SetMultipartFields(fields ...*MultipartField)`? I think it would convenient to use.